src/README.build.md

```diff
@@ -0,0 +1,55 @@
+# Standalone WRF-Hydro Build Instructions
+Details regarding the model as well as documentation and user guides can be
+found on the project website:
+https://ral.ucar.edu/projects/wrf_hydro
+
+## Requirements
+* A supported Fortran compiler:
+  - GNU (gfortran)
+  - Cray (ftn)
+  - Intel (ifort)
+  - NVidia (nvfortran)
+* MPI libraries (MPICH or Open MPI)
```

Comment:
We also support Intel MPI -- could add that, or just put ", etc." since I think pretty much any standards-compliant MPI library should work.

Reply:
Good point! I'll add `etc.`